backend/__tests__/userReview.test.js

```
@@ -0,0 +1,326 @@
+        await user.save();
+        
+        await sendNotification(
+          property.userID,
+          `${user.username} has reviewed your property with a rating of ${rating} and commented: "${comment}"`,
+          'review'
+        );
+        
+        return res.status(200).json({ message: 'Review added' });
+      } catch (error) {
+        // Simply rethrow the error for testing error handling
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)